com.transmissionbt.Transmission.json

```diff
@@ -18,14 +19,18 @@
         "--talk-name=org.freedesktop.Notifications",
         "--filesystem=xdg-run/gvfsd",
         "--talk-name=org.gtk.vfs.*",
-        "--talk-name=org.kde.StatusNotifierWatcher"
+        "--talk-name=org.kde.StatusNotifierWatcher",
+        "--own-name=com.transmissionbt.*"
```

Comment:
I previously had a patch to make this unnecessary. It was dropped in this update. Why? It should be pretty easy to refresh it.

Reply:
The file (gtk/main.c) doesn't exist anymore. I can recreate that if you want.